Sources/StreamChatUI/MessageActionsPopup/ChatMessagePopupVC.swift

```diff
@@ -62,17 +60,19 @@ open class _ChatMessagePopupVC<ExtraData: ExtraDataTypes>: _ViewController, Comp
     }
 
     override open func setUpLayout() {
-        scrollView.embed(scrollContentView)
+        guard messageViewFrame != nil else { return }
```

Comment:
We shouldn't silently fail here but log an error with a description of what's happening instead.